lib/models/tree-options.model.ts

```diff
@@ -98,6 +96,14 @@ export class TreeOptions {
       deprecated('hasCustomContextMenu', 'actionMapping: mouse: contextMenu');
     }
 
+    if (options.treeNodeTemplate) {
+      deprecated('treeNodeTemplate', 'a template in the content of the <Tree> component like this: <Tree><template #treeNodeTemplate let-node>...</template></Tree>.  If you don\'t have time to update your code and don\'t need AoT compilation, use DeprecatedTreeModule');
+    }
+
+    if (options.loadingComponent) {
+      deprecated('loadingComponent', 'a template in the content of the <Tree> component like this: <Tree><template #loadingTemplate>...</template></Tree>.  If you don\'t have time to update your code and don\'t need AoT compilation, use DeprecatedTreeModule');
+    }
+
```

Comment:
Now that we're working with templates we should also deprecate `options.context`.